docs/source/restore/index.rst

```diff
@@ -85,3 +85,20 @@ Restore speed is controlled by many parameters (see :ref:`sctool restore <sctool
 The ``--unpin-agent-cpu`` is disabled by default, but in case you observe small download
 bandwidth, you could try to :ref:`pause <task-stop>` restore task, :ref:`update <restore-update>` it with ``--unpin-agent-cpu``,
 and :ref:`resume <task-start>` it.
+
+Interference with backup and repair tasks
+=========================================
+
+Starting from Scylla Manager v3.5.0, the restore task will not start if backup or repair tasks are running in the cluster at that time, and vice versa, if backup or repair tasks are running, the restore task will not start.
+
+Prior to 3.5.0, it is advisable to manually make sure that the restore task does not interfere with backup or repair tasks.
```

Comment:
SM docs are versioned per minor version, so there is no need to write things like "starting from version".
On the other hand, I see that you wanted to document previous behavior as well.
So the proper way of handling this would be to create 2 separate PRs:
- one adding docs about current state (merged to master and branch-3.4)
- one updating docs with new functionality description (rebased on the first one, merged to master)

Reply:
yes, that make sense to me! I've split this pr into two part. This one is related to documenting new behavior and #4147 about current(old) behavior

Reply:
Thanks for the additional effort!